### ISSUE TITLE
[upmeter]  upmeter-probe-controller-manager is not removed for a long time 

### ISSUE THREAD

**Issue by alexey-gavrilov-flant** [deckhouse/deckhouse#6672]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Version

v1.54.5

### Expected Behavior

tests should not stay for long

### Actual Behavior

upmeter-probe-controller-manager sometimes stays and is not removed.
new upmeter-probe-controller-managers are created in parallel.

### Logs

```shell
upmeter-probe-controller-manager-acebfbff   0/1     3d12h
```

**Comment by konstantin-axenov** [comment]
Closed as duplicate of https://github.com/deckhouse/deckhouse/issues/6672

**Comment by alexey-gavrilov-flant** [comment]
https://github.com/deckhouse/deckhouse/issues/2327
